packages/grid/_modules_/grid/hooks/features/rows/useGridEditRows.ts

```diff
@@ -333,10 +355,11 @@ export function useGridEditRows(apiRef: GridApiRef) {
       getCellMode,
       isCellEditable,
       commitCellChange,
-      setEditCellProps,
+      setEditCellProps, // TODO don't expose, update the editRowsModel prop directly
```

Comment:
why this comment?

Reply:
To reduce confusion between `setEditCellValue`. In edit components, the current value is only thing that needs to be updated in the majority of the cases. Outside an edit components, developers can control the state with the `editRowsModel` prop to set the error or other prop.